src/SassBuilder.php

```diff
@@ -26,6 +26,7 @@ public function __construct(
         private readonly string $projectRootDir,
         private readonly ?string $binaryPath,
         private readonly bool $embedSourcemap,
+        private readonly array $loadPaths,
```

Comment:
I wonder if it will lead to BC breaks if we release this feature as a minor version bump?

Reply:
Hmm yes it could. Would default to an empty array be enough to handle this?

Reply:
Yeah, I think setting it to an empty array should do the trick. Probably we could add a docblock comment explaining it's default to an empty array just for BC

Reply:
I added the default value and related docblock entry